contrib/platform/src/com/sun/jna/platform/win32/Winspool.java

```diff
@@ -571,6 +661,30 @@ boolean OpenPrinter(
             HANDLEByReference phPrinter,
             // _In_
             LPPRINTER_DEFAULTS pDefault);
+    
+    boolean GetForm(
```

Comment:
I tried to match the formatting with the rest of the file (i.e. OpenPrinter).  I apologize if this is not what you wanted.